<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1607</width>
    <height>1017</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="addfu">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>90</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Function</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>140</y>
     <width>211</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="script">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>140</y>
     <width>771</width>
     <height>791</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>80</y>
     <width>170</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Function</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>40</y>
     <width>1621</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>70</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>70</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>141</y>
     <width>461</width>
     <height>791</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Explorer</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="stop_5">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>80</y>
     <width>170</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add ScreenShot</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
